samples/net/sockets/http_server/sample.yaml

```diff
@@ -0,0 +1,16 @@
+sample:
+  description: HTTP Server Sample
+  name: http_server_sample
+common:
+  harness: net
+  min_ram: 32
+  tags:
+    - http
+    - net
+    - server
+    - socket
+  integration_platforms:
+    - native_posix_64
```

Comment:
I would remove this for now. The assumption that the app could build against native_posix was based on word-of-mouth statement from the POSIX arch maintainer that POSIX API could work on ARCH POSIX, but that is certainly not the case.